packages/flutter/lib/src/cupertino/page_scaffold.dart

```diff
@@ -22,6 +22,7 @@ class CupertinoPageScaffold extends StatelessWidget {
   const CupertinoPageScaffold({
     Key key,
     this.navigationBar,
+    this.backgroundColor,
```

Comment:
default to CupertinoColors.white up here (so users can explicitly specify `null` and get a transparent scaffold if they want)